pkg/plugins/policies/meshhttproute/plugin/v1alpha1/listeners.go

```diff
@@ -146,8 +146,26 @@ func prepareRoutes(
 
 	var routes []Route
 	for _, rule := range rules {
+		var matches []api.Match
+
+		for _, match := range rule.Matches {
+			if match.Path == nil {
+				// According to Envoy docs, match must have precisely one of
+				// prefix, path, safe_regex, connect_matcher,
+				// path_separated_prefix, path_match_policy set, so when policy
+				// doesn't specify explicit type of matching, we are assuming
+				// "catch all" path (any path starting with "/").
+				match.Path = &api.PathMatch{
+					Value: "/",
```

Comment:
nit: could be defined privately in this file and used here and above at `catchAllMatch`

Reply:
addressed